### ISSUE TITLE
Wrong reference to SSL profile

### ISSUE THREAD

**Issue by makagonr** [f5devcentral/f5-as3-config-converter#25]
### Environment
I have problem with 1.12 and 1.11 version of the ACC and possibly also previous ones.

Converting [bigip.conf.txt](https://github.com/f5devcentral/f5-as3-config-converter/files/6522635/bigip.conf.txt)

Results with a Virtual server with wrong reference to the profile.

Inside the configuration, VS is in partition "tenant2" and the SSL profile is placed inside /Common/Shared tenant.

Unfortunately, the reference from VS to the profile does not include "/Common/Shared" prefix:

```
    "tenant_2": {                                                                                                                           
        "class": "Tenant",                                                                                                                  
        "application_2": {                                                                                                                  
            "class": "Application",                                                                                                         
            "template": "generic",                                                                                                          
            "VS_app2": {                                                                                                                    
                ...
                "class": "Service_TCP",                                                                                                     
                "serverTLS": "custom_ssl_client_profile",                                                                                   
                "clientTLS": "custom_ssl_server_profile",               
 ...
   "Common": {                                                                                                                             
        "class": "Tenant",                                                                                                                  
        "Shared": {                                                                                                                         
            "class": "Application",                                                                                                         
...
            "custom_ssl_client_profile": {                                                                                                  
                "certificates": [                                                                                                           
...
            },                                                                                                                              
            "custom_ssl_server_profile": {                                                                                                  
                "clientCertificate": [                                                                                                      
...
```
Expected behaviour would be to reference serverTLS and clientTLS with full prefix:
```
    "tenant_2": {                                                                                                                           
        "class": "Tenant",                                                                                                                  
        "application_2": {                                                                                                                  
            "class": "Application",                                                                                                         
            "template": "generic",                                                                                                          
            "VS_app2": {                                                                                                                    
                ...
                "class": "Service_TCP",                                                                                                     
                "serverTLS": "/Common/Shared/custom_ssl_client_profile",                                                                                   
                "clientTLS": "/Common/Shared/custom_ssl_server_profile",               
 ...
   "Common": {                                                                                                                             
        "class": "Tenant",                                                                                                                  
        "Shared": {                                                                                                                         
            "class": "Application",                                                                                                         
...
            "custom_ssl_client_profile": {                                                                                                  
                "certificates": [                                                                                                           
...
            },                                                                                                                              
            "custom_ssl_server_profile": {                                                                                                  
                "clientCertificate": [                                                                                                      
...
```

**Comment by mdditt2000** [comment]
@makagonr everything is good except for 

```
"custom_ssl_client_profile": {
                "certificates": [
                    {
                        "certificate": "/Common/Shared/custom_ssl_cert_key"
                    }
                ],
                "class": "TLS_Server",
                "insertEmptyFragmentsEnabled": true
            },
```

I am going to request DEV remove the /Common/Shared/. Should be

                "certificates": [
                    {
                        "certificate": "/Common/custom_ssl_cert_key"
                    }

**Comment by mdditt2000** [comment]
Add CHARON-300 in in-play for PD to pickup. Possible ACC 1.13 release. Please watch the milestone

**Comment by makagonr** [comment]
@mdditt2000 This reference is to AS3 object, so it needs to be "/Common/Shared/custom_ssl_cert_key", the certificate object itself, can refer to bigip cert object using "bigip": "/Common/cert..."

**Comment by mdditt2000** [comment]
Issue resolved in ACC 1.13
